recipes/iterative-telemetry/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "iterative-telemetry" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/iterative/telemetry-python/archive/{{ version }}.tar.gz
```

Comment:
Used github url here instead of pypi because of https://github.com/conda-forge/staged-recipes/pull/19760#issuecomment-1193001480 . Same for version 0.0.5, not sure why this is happening (might be related to the github repo and package name mismatch or the fact that we've renamed the github repo as well). In any case, this seems to work now and is reliable for now.

Reply:
Oh wow, it does work now. So it was some hickup on pypi after all. Adjusting...